stream_alert_cli/version.py

```diff
@@ -106,30 +104,53 @@ def _publish_helper(self, **kwargs):
         client = boto3.client('lambda', region_name=region)
         code_sha_256 = self.config['lambda'][self.package.config_key]['source_current_hash']
 
-        # Publish the function
+        # Publish the function(s)
+        if self.package.package_name == 'stream_alert_app':
+            for app_name, app_info in self.config['clusters'][cluster]['modules'] \
+                ['stream_alert_apps'].iteritems():
+                # Name follows format: '<prefix>_<cluster>_<service>_<app_name>_app'
+                function_name = '_'.join([self.config['global']['account']['prefix'], cluster,
+                                          app_info['type'], app_name, 'app'])
+                new_version = self._publish(client, function_name, code_sha_256)
+                if not new_version:
```

Comment:
why wouldn't it be a new version?

Reply:
The `_version_helper` method that this calls is within a try/except for a boto3 call. on exception, it will return False: https://github.com/airbnb/streamalert/pull/323/files/91d7ef4c86983517af3119584bf5462c1cc87b4f..d465fc724eebf0f838e8750cc1d787b0b5f8f592#diff-106ec47af4d3b5071339461a157d8e14R74

This _should not_ happen, but I noticed that when trying to have app integrations pre-configured (before the `terraform init` happens) this would fail and return False due to not being able to find the function in S3. After speaking with you, that error is obviously expected, but adding this check seemed like it would still be a good idea. Thoughts?